discovery-provider/src/api_helpers.py

```diff
@@ -61,18 +52,21 @@ def response_dict_with_metadata(response_entity=None):
     response_dictionary['latest_indexed_block'] = (int(latest_indexed_block) if latest_indexed_block else None)
     response_dictionary['latest_chain_block'] = (int(latest_chain_block) if latest_chain_block else None)
     response_dictionary['version'] = disc_prov_version
-    response_dictionary['owner_wallet'] = shared_config['delegate']['owner_wallet']
-
-    return response_dictionary
+    response_dictionary['delegateWallet'] = shared_config['delegate']['owner_wallet']
```

Comment:
@vicky-g we should make this consistent case

Reply:
i think we should leave `delegateWallet` as camel case, so when we recover the wallet in libs, we can access the value

https://github.com/AudiusProject/audius-protocol/pull/586/files#diff-101c0411bfa295318bcea2fe0be04050R18

Reply:
@vicky-g this is evading the issue, but why not just make it wallet? signature is the hash that's been signed by wallet

Reply:
@roneilr we wanted to rename this key to be more specific?

Reply:
imo "wallet" is fine - it could be a duplicate of the primary on-chain wallet or it can be the "delegate" if one elects to fill that in

Reply:
👍  lets go with "delegate" to avoid any potential conflicts on chain. @dmanjunath is this ok with you?